src/controllers/posts.js

```diff
@@ -21,3 +21,26 @@ export const createPost = async (req, res) => {
     });
   }
 };
+
+export const createComment = async (req, res) => {
+  const { comment_content } = req.body;
+
+  const { id: post_id } = req.params;
```

Comment:
Need to check whether the post id is valid and not null, otherwise server crashes